### ISSUE TITLE
Error: SyntaxError: Failed to execute 'open' on 'XMLHttpRequest': '[object Arguments]'

### ISSUE THREAD

**Issue by albacoretuna** [Leanplum/Leanplum-JavaScript-SDK#82]
## Expected Behavior
No errors 

## Actual Behavior
```
SyntaxError: Failed to execute 'open' on 'XMLHttpRequest': '[object Arguments]' is not a vali
```

## Steps to Reproduce the Problem

Upgrade to 1.5 and open our website, for a user that has many variables that are not initialized in the javascript. 

Might be somehow similar to #22 

## Root cause and a possible fix 
[This line ](https://github.com/Leanplum/Leanplum-JavaScript-SDK/blob/1b3a66a2fca9ec715cc89a82f7a9cc49f6111537/src/Network.ts#L199) now it reads:
 
```javascript
this.ajax.call(this, args)
```

But it used to be

```javasript
      apply.call(Network.ajax, null, args)
```

I guess the current version should become 

```javascript 
this.ajax.apply(this, args)
```

## Specifications

  - Version: 1.5
  - Platform: all

**Comment by iignatov** [comment]
@albacoretuna Thanks for reporting this! Your assumptions are correct, it seems that this got broken after a recent refactoring. It should be fixed in the next version which should be released in the next couple of days.

**Comment by gyoshev** [comment]
@albacoretuna the fix is included in version `1.6.0`, available on NPM.

**Comment by albacoretuna** [comment]
Great. Thanks
